028-HubAndSpoke/Student/README.md

```diff
@@ -0,0 +1,30 @@
+# What the Hack - Azure Networking Hub and Spoke
```

Comment:
Please merge this content with the top level README.md file and remove this file.

Remember that students start at the root of your hack for the overview information and then navigate to challenges from there. This is more noticeable when you see it rendered. Go to: aka.ms/wthhome and then click into "Intro to Kubernetes"

Reply:
Done!